tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -1207,7 +1241,44 @@ ROOT::Experimental::Internal::RMiniFileReader::GetNTupleBare(std::string_view nt
 
 void ROOT::Experimental::Internal::RMiniFileReader::ReadBuffer(void *buffer, size_t nbytes, std::uint64_t offset)
 {
-   auto nread = fRawFile->ReadAt(buffer, nbytes, offset);
+   std::uint64_t maxBlobSize = fMaxBlobSize;
```

Comment:
Saving it in a local potentially avoids extra loads that the compiler may insert due to the many functions with side-effects on the `this` object...although it is arguably a negligible win since we're doing lots of I/O here.